### PR TITLE
Base blocks and items, and datagen stuff

### PR DESCRIPTION
**Pull request by IchHabeHunger54**
This PR adds the basic crafting and minor functionality blocks and items. NYI: the Crystal Phylactery, the flowers and the book.

Also provides the data providers and helper methods for most of them.

### CODE REVIEW

**Comment by IchHabeHunger54** [comment]
Merge-ready. Crystal Phylactery is to be added in another PR.

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMAdvancementProvider.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+     * @param parent   The id of this advancement's parent.
+     * @param item     The item to display.
+     * @param type     The frame type of this advancement.
+     * @param toast    Whether to show a toast in the top right corner or not.
+     * @param chat     Whether to announce the advancement to chat or not.
+     * @param hide     Whether the advancement should be visible before unlocking or not.
+     * @param criteria Optional advancement criteria. Should be made through Pair.of("id", criterion);
+     */
+    static Advancement addAdvancement(Consumer<Advancement> consumer, String folder, String id, Advancement parent, ItemLike item, FrameType type, boolean toast, boolean chat, boolean hide, Pair<String, CriterionTriggerInstance>... criteria) {
+        Advancement.Builder builder = Advancement.Builder.advancement().parent(parent).display(item, new TranslatableComponent("advancements." + ArsMagicaAPI.MOD_ID + "." + id + ".title"), new TranslatableComponent("advancements." + ArsMagicaAPI.MOD_ID + "." + id + ".description"), null, type, toast, chat, hide);
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMAdvancementProvider.java]: use `Util.makeDescriptionId()` instead
```suggestion
        Advancement.Builder builder = Advancement.Builder.advancement().parent(parent).display(item, new TranslatableComponent(Util.makeDescriptionId("advancement", new ResourceLocation(ArsMagicaAPI.MOD_ID, id))+".title"), new TranslatableComponent(Util.makeDescriptionId("advancement", new ResourceLocation(ArsMagicaAPI.MOD_ID, id))+".description"), null, type, toast, chat, hide);
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMAdvancementProvider.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+     * @param parent   The id of this advancement's parent.
+     * @param item     The ItemStack to display.
+     * @param type     The frame type of this advancement.
+     * @param toast    Whether to show a toast in the top right corner or not.
+     * @param chat     Whether to announce the advancement to chat or not.
+     * @param hide     Whether the advancement should be visible before unlocking or not.
+     * @param criteria Optional advancement criteria. Should be made through Pair.of("id", criterion);
+     */
+    static Advancement addAdvancement(Consumer<Advancement> consumer, String folder, String id, Advancement parent, ItemStack item, FrameType type, boolean toast, boolean chat, boolean hide, Pair<String, CriterionTriggerInstance>... criteria) {
+        Advancement.Builder builder = Advancement.Builder.advancement().parent(parent).display(item, new TranslatableComponent("advancements." + ArsMagicaAPI.MOD_ID + "." + id + ".title"), new TranslatableComponent("advancements." + ArsMagicaAPI.MOD_ID + "." + id + ".description"), null, type, toast, chat, hide);
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMAdvancementProvider.java]: here too
```suggestion
        Advancement.Builder builder = Advancement.Builder.advancement().parent(parent).display(item, new TranslatableComponent(Util.makeDescriptionId("advancement", new ResourceLocation(ArsMagicaAPI.MOD_ID, id))+".title"), new TranslatableComponent(Util.makeDescriptionId("advancement", new ResourceLocation(ArsMagicaAPI.MOD_ID, id))+".description"), null, type, toast, chat, hide);
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMAdvancementProvider.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        return builder.save(consumer, ArsMagicaAPI.MOD_ID + ":" + folder + "/" + id);
+    }
+
+    /**
+     * Contains all normal advancements.
+     */
+    private static final class AMAdvancements implements Consumer<Consumer<Advancement>> {
+        @Override
+        public void accept(Consumer<Advancement> consumer) {
+            Advancement root = Advancement.Builder.advancement().display(ArsMagicaAPI.get().getBookStack(), new TranslatableComponent(Util.makeDescriptionId("advancements", new ResourceLocation(ArsMagicaAPI.MOD_ID, "root/title"))), new TranslatableComponent(Util.makeDescriptionId("advancements", new ResourceLocation(ArsMagicaAPI.MOD_ID, "root/description"))), new ResourceLocation(ArsMagicaAPI.MOD_ID, "textures/gui/advancements/backgrounds/arsmagicalegacy.png"), FrameType.TASK, false, false, false).addCriterion("arcane_compendium", InventoryChangeTrigger.TriggerInstance.hasItems(ArsMagicaAPI.get().getBookStack().getItem())).save(consumer, ArsMagicaAPI.MOD_ID + ":root");
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMAdvancementProvider.java]:
```suggestion
            Advancement root = Advancement.Builder.advancement().display(ArsMagicaAPI.get().getBookStack(), new TranslatableComponent(Util.makeDescriptionId("advancement", new ResourceLocation(ArsMagicaAPI.MOD_ID, "root/title"))), new TranslatableComponent(Util.makeDescriptionId("advancement", new ResourceLocation(ArsMagicaAPI.MOD_ID, "root/description"))), new ResourceLocation(ArsMagicaAPI.MOD_ID, "textures/gui/advancements/backgrounds/arsmagicalegacy.png"), FrameType.TASK, false, false, false).addCriterion("arcane_compendium", InventoryChangeTrigger.TriggerInstance.hasItems(ArsMagicaAPI.get().getBookStack().getItem())).save(consumer, ArsMagicaAPI.MOD_ID + ":root");
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+    /**
+     * Adds a block translation.
+     *
+     * @param block       The block to generate the translation for.
+     * @param translation The translation to apply.
+     */
+    private void blockTranslation(Supplier<? extends Block> block, String translation) {
+        addBlock(block, translation);
+    }
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]: Not needed

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+    /**
+     * Adds an item translation.
+     *
+     * @param item        The item to generate the translation for.
+     * @param translation The translation to apply.
+     */
+    private void itemTranslation(Supplier<? extends Item> item, String translation) {
+        addItem(item, translation);
+    }
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]: Not needed

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+
+    /**
+     * Adds an item translation that matches the item id.
+     *
+     * @param item The item to generate the translation for.
+     */
+    private void itemIdTranslation(Supplier<? extends Item> item) {
+        addItem(item, idToTranslation(item.get().getRegistryName().getPath()));
+    }
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]: use RegistryObject directly
```suggestion
    private void itemIdTranslation(RegistryObject<? extends Item> item) {
        addItem(item, idToTranslation(item.getId().getPath()));
    }
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+
+    /**
+     * Adds an effect translation that matches the effect id.
+     *
+     * @param effect The effect to generate the translation for.
+     */
+    private void effectIdTranslation(Supplier<? extends MobEffect> effect) {
+        addEffect(effect, effect.get().getRegistryName().getPath());
+    }
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]: - forgot call to idToTranslation
- use RegistryObject directly
```suggestion
    private void effectIdTranslation(RegistryObject<? extends MobEffect> effect) {
        addEffect(effect, idToTranslation(effect.getId().getPath()));
    }
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+
+    /**
+     * Adds an enchantment translation that matches the enchantment id.
+     *
+     * @param enchantment The enchantment to generate the translation for.
+     */
+    private void enchantmentIdTranslation(Supplier<? extends Enchantment> enchantment) {
+        addEnchantment(enchantment, enchantment.get().getRegistryName().getPath());
+    }
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]: - forgot call to idToTranslation
- use RegistryObject directly
```suggestion
    private void enchantmentIdTranslation(RegistryObject<? extends Enchantment> enchantment) {
        addEnchantment(enchantment, idToTranslation(enchantment.getId().getPath()));
    }
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+
+    /**
+     * Adds an entity translation that matches the entity id.
+     *
+     * @param entity The entity to generate the translation for.
+     */
+    private void entityIdTranslation(Supplier<? extends EntityType<?>> entity) {
+        addEntityType(entity, entity.get().getRegistryName().getPath());
+    }
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]: - forgot call to idToTranslation
- use RegistryObject directly
```suggestion
    private void entityIdTranslation(RegistryObject<? extends EntityType<?>> entity) {
        addEntityType(entity, idToTranslation(entity.getId().getPath()));
    }
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+     * Adds an advancement translation.
+     *
+     * @param advancement The advancement id.
+     * @param title       The advancement title.
+     * @param description The advancement description.
+     */
+    private void advancementTranslation(String advancement, String title, String description) {
+        add("advancements." + ArsMagicaAPI.MOD_ID + "." + advancement + ".title", title);
+        add("advancements." + ArsMagicaAPI.MOD_ID + "." + advancement + ".description", description);
+    }
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]: use Util.makeDescriptionId() and ResourceLocation 
```suggestion
    private void advancementTranslation(ResourceLocation advancement, String title, String description) {
        add(Util.makeDescriptionId("advancement", advancement) + ".title", title);
        add(Util.makeDescriptionId("advancement", advancement) + ".description", description);
    }
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+
+    /**
+     * Adds a translation.
+     *
+     * @param key         The translation key to use.
+     * @param translation The translation to apply.
+     */
+    private void translate(String key, String translation) {
+        add(key, translation);
+    }
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]: Not needed

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+        itemIdTranslation(AMItems.ARCANE_ASH);
+        itemIdTranslation(AMItems.PURIFIED_VINTEUM_DUST);
+        blockIdTranslation(AMBlocks.AUM);
+        blockIdTranslation(AMBlocks.CERUBLOSSOM);
+        blockIdTranslation(AMBlocks.DESERT_NOVA);
+        blockIdTranslation(AMBlocks.TARMA_ROOT);
+        blockIdTranslation(AMBlocks.WAKEBLOOM);
+        blockIdTranslation(AMBlocks.VINTEUM_TORCH);
+        blockTranslation(AMBlocks.WIZARDS_CHALK, "Wizard's Chalk");
+        advancementTranslation("root", ArsMagicaLegacy.getModName(), "A renewed look into Minecraft with a splash of magic...");
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMEnglishLanguageProvider.java]:
```suggestion
        advancementTranslation(new ResourceLocation(ArsMagicaAPI.MOD_ID, "root"), ArsMagicaLegacy.getModName(), "A renewed look into Minecraft with a splash of magic...");
```

File: src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMPatchouliBookProvider.java [hunk 13]
```diff
@@ -1,35 +1,43 @@
...
-                .setModel(new ResourceLocation(ArsMagicaAPI.MOD_ID, "arcane_compendium"))
-                .setCreativeTab(ArsMagicaAPI.get().getItemGroup().getRecipeFolderName())
-                .setUseResourcepack()
-                .addCategory("test", "TestCategory", "This is a test", new ItemStack(Items.BARRIER))
-                    .addEntry("test", "TestEntry", new ItemStack(Items.GLOW_BERRIES))
-                        .addSimpleTextPage("Lorem ipsum dolor sit")
-                    .build()
-                .build()
+    protected void addBooks(Consumer<BookBuilder<?, ?, ?>> consumer) {
+        createBookBuilder("arcane_compendium", "Arcane Compendium", "A renewed look into Minecraft with a splash of magic...", lang).setSubtitle("1st Edition").setModel(new ResourceLocation(ArsMagicaAPI.MOD_ID, "arcane_compendium")).setCreativeTab(ArsMagicaAPI.get().getItemGroup().getRecipeFolderName()).setUseResourcepack()
```
Comment (Minecraftschurli) [inline src/data/java/com/github/minecraftschurli/arsmagicalegacy/data/AMPatchouliBookProvider.java]: 1. Stop the f***ing one liner
2. use version for numbered book editions
```suggestion
        createBookBuilder("arcane_compendium", "Arcane Compendium", "A renewed look into Minecraft with a splash of magic...", lang)
        .setVersion("1")
        .setModel(new ResourceLocation(ArsMagicaAPI.MOD_ID, "arcane_compendium"))
        .setCreativeTab(ArsMagicaAPI.get().getItemGroup().getRecipeFolderName())
        .setUseResourcepack()
```

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/init/AMBlocks.java [hunk 14]
```diff
@@ -1,7 +1,88 @@
...
+    RegistryObject<FlowerBlock> WAKEBLOOM = BLOCKS.register("wakebloom", () -> new FlowerBlock(/*AMEffects.BURNOUT_REDUCTION.get()*/MobEffects.MOVEMENT_SLOWDOWN, 7, BlockBehaviour.Properties.copy(Blocks.POPPY)));
+    RegistryObject<TorchBlock> VINTEUM_TORCH = BLOCKS.register("vinteum_torch", () -> new TorchBlock(BlockBehaviour.Properties.of(Material.DECORATION).noCollission().instabreak().lightLevel((p_50886_) -> 14).sound(SoundType.WOOD), ParticleTypes.SMOKE));
+    RegistryObject<WallTorchBlock> VINTEUM_WALL_TORCH = BLOCKS.register("vinteum_wall_torch", () -> new WallTorchBlock(BlockBehaviour.Properties.of(Material.DECORATION).noCollission().instabreak().lightLevel((p_50886_) -> 14).sound(SoundType.WOOD), ParticleTypes.SMOKE));
+    RegistryObject<WizardsChalkBlock> WIZARDS_CHALK = BLOCKS.register("wizards_chalk", () -> new WizardsChalkBlock(BlockBehaviour.Properties.of(Material.DECORATION).noCollission().sound(SoundType.GRAVEL)));
+
+    /**
+     * Empty method that is required for classloading
+     */
+    static void init() {}
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/init/AMBlocks.java]:
```suggestion
    @Internal
    static void register() {}
```

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/init/AMContainers.java [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+
+import static com.github.minecraftschurli.arsmagicalegacy.common.init.AMRegistries.CONTAINERS;
+
+public interface AMContainers {
+    RegistryObject<MenuType<RuneBagContainer>> RUNE_BAG = CONTAINERS.register("rune_bag", () -> IForgeContainerType.create((windowId, inv, data) -> new RuneBagContainer(windowId, inv, inv.player.getItemInHand(data.readBoolean() ? InteractionHand.MAIN_HAND : InteractionHand.OFF_HAND))));
+
+    /**
+     * Empty method that is required for classloading
+     */
+    static void init() {}
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/init/AMContainers.java]:
```suggestion
    @Internal
    static void register() {}
```

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/init/AMItems.java [hunk 16]
```diff
@@ -19,8 +27,81 @@ public ItemStack makeIcon() {
...
+    RegistryObject<BlockItem> DESERT_NOVA = ITEMS.register("desert_nova", () -> new BlockItem(AMBlocks.DESERT_NOVA.get(), ITEM_64));
+    RegistryObject<BlockItem> TARMA_ROOT = ITEMS.register("tarma_root", () -> new BlockItem(AMBlocks.TARMA_ROOT.get(), ITEM_64));
+    RegistryObject<BlockItem> WAKEBLOOM = ITEMS.register("wakebloom", () -> new BlockItem(AMBlocks.WAKEBLOOM.get(), ITEM_64));
+    RegistryObject<StandingAndWallBlockItem> VINTEUM_TORCH = ITEMS.register("vinteum_torch", () -> new StandingAndWallBlockItem(AMBlocks.VINTEUM_TORCH.get(), AMBlocks.VINTEUM_WALL_TORCH.get(), ITEM_64));
+    RegistryObject<WizardsChalkItem> WIZARDS_CHALK = ITEMS.register("wizards_chalk", () -> new WizardsChalkItem(ITEM_64.durability(100)));
+
+    /**
+     * Empty method that is required for classloading
+     */
+    static void init() {}
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/init/AMItems.java]:
```suggestion
    @Internal
    static void register() {}
```

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/init/AMRegistries.java [hunk 17]
```diff
@@ -41,6 +41,9 @@
      */
     @Internal
     static void init(IEventBus bus) {
+        AMBlocks.init();
+        AMItems.init();
+        AMContainers.init();
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/init/AMRegistries.java]:
```suggestion
        AMBlocks.register();
        AMItems.register();
        AMContainers.register();
```

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagContainer.java [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+                    public boolean mayPlace(@NotNull ItemStack pStack) {
+                        return pStack.getItem() instanceof ColoredRuneItem && k == ((ColoredRuneItem) pStack.getItem()).getDyeColor().getId();
+                    }
+                });
+            }
+        for (int i = 0; i < 3; i++)
+            for (int j = 0; j < 9; j++)
+                addSlot(new Slot(playerInventory, i * 9 + j + 9, 8 + j * 18, 68 + i * 18));
+        for (int i = 0; i < 9; i++)
+            addSlot(new Slot(playerInventory, i, 8 + i * 18, 126));
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagContainer.java]: for loops need `{}`

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagContainer.java [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+            is = stack.copy();
+            if (pIndex < 16) {
+                if (!moveItemStackTo(stack, 16, 52, true)) return ItemStack.EMPTY;
+            } else if (stack.getItem() instanceof ColoredRuneItem) {
+                int color = ((ColoredRuneItem) stack.getItem()).getDyeColor().getId();
+                Slot invSlot = slots.get(color);
+                if (invSlot.mayPlace(is) && !moveItemStackTo(stack, color, color + 1, true)) return ItemStack.EMPTY;
+            }
+            if (stack.isEmpty()) slot.set(ItemStack.EMPTY);
+            else slot.setChanged();
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagContainer.java]: if with else needs `{}`

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagInventory.java [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+ * {@see https://github.com/VazkiiMods/Botania/blob/master/src/main/java/vazkii/botania/common/item/ItemBackedInventory.java}
+ */
+public class RuneBagInventory extends SimpleContainer {
+    private final ItemStack stack;
+
+    public RuneBagInventory(ItemStack stack) {
+        super(DyeColor.values().length);
+        this.stack = stack;
+        ListTag tag = !stack.isEmpty() && stack.getOrCreateTag().contains("Items") ? stack.getOrCreateTag().getList("Items", 10) : new ListTag();
+        for (int i = 0; i < DyeColor.values().length && i < tag.size(); i++) setItem(i, ItemStack.of(tag.getCompound(i)));
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagInventory.java]: for loops need `{}`

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagInventory.java [hunk 21]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    @Override
+    public void setChanged() {
+        super.setChanged();
+        ListTag tag = new ListTag();
+        for (int i = 0; i < DyeColor.values().length; i++) tag.add(getItem(i).save(new CompoundTag()));
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagInventory.java]: for loops need `{}`

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagInventory.java [hunk 22]
```diff
@@ -0,0 +1,30 @@
...
+import net.minecraft.nbt.ListTag;
+import net.minecraft.world.SimpleContainer;
+import net.minecraft.world.item.DyeColor;
+import net.minecraft.world.item.ItemStack;
+
+/**
+ * Mostly taken from the Botania mod.
+ * {@see https://github.com/VazkiiMods/Botania/blob/master/src/main/java/vazkii/botania/common/item/ItemBackedInventory.java}
+ */
+public class RuneBagInventory extends SimpleContainer {
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagInventory.java]: Naming convention

File: src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagContainer.java [hunk 23]
```diff
@@ -0,0 +1,68 @@
...
+import net.minecraft.world.inventory.Slot;
+import net.minecraft.world.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Mostly taken from McJty's tutorials and the Botania mod.
+ * {@see https://github.com/McJty/YouTubeModding14/blob/1.17/src/main/java/com/mcjty/mytutorial/blocks/FirstBlockContainer.java}
+ * {@see https://github.com/VazkiiMods/Botania/blob/master/src/main/java/vazkii/botania/client/gui/bag/ContainerFlowerBag.java}
+ */
+public class RuneBagContainer extends AbstractContainerMenu {
```
Comment (Minecraftschurli) [inline src/main/java/com/github/minecraftschurli/arsmagicalegacy/common/item/runebag/RuneBagContainer.java]: Naming convention

**Review by Minecraftschurli — CHANGES_REQUESTED** [review]
Looks good apart from:

**Comment by IchHabeHunger54** [comment]
Changes were applied.

**Review by Minecraftschurli — APPROVED** [review]
Looks good now

**Pull request merged**